bug: build failed on target x86_64 with avx2

Seems the wrong function name is called here, probably a typo.

```
error[E0425]: cannot find function `vec_dot_f32_f32_strided_simd` in this scope
   --> crabml-core/src/backends/cpu/buf/buf_f32.rs:54:9
    |
54  |         vec_dot_f32_f32_strided_simd(a, a_base, a_stride, k, b)
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: a function with a similar name exists: `vec_dot_f32_strided_simd`
...
135 | fn vec_dot_f32_strided_simd(a: &[f32], a_base: usize, a_stride: usize, k: usize, b: &[f32]) -> f32 {
    | -------------------------------------------------------------------------------------------------- similarly named function `vec_dot_f32_strided_simd` defined here

For more information about this error, try `rustc --explain E0425`.
error: could not compile `crabml` (lib) due to 1 previous error
warning: build failed, waiting for other jobs to finish...
```

--- a/crabml-core/src/backends/cpu/buf/buf_f32.rs
+++ b/crabml-core/src/backends/cpu/buf/buf_f32.rs
@@ -132,7 +132,13 @@ fn vec_dot_f32_f32_strided_simd(
 
 #[cfg(target_arch = "x86_64")]
 #[cfg(target_feature = "avx2")]
-fn vec_dot_f32_f32_strided_simd(a: &[f32], a_base: usize, a_stride: usize, k: usize, b: &[f32]) -> f32 {
+fn vec_dot_f32_f32_strided_simd(
+    a: &[f32],
+    a_base: usize,
+    a_stride: usize,
+    k: usize,
+    b: &[f32],
+) -> f32 {
     use std::arch::x86_64::*;
 
     unsafe {

--- a/crabml-core/src/backends/cpu/buf/buf_f32.rs
+++ b/crabml-core/src/backends/cpu/buf/buf_f32.rs
@@ -132,7 +132,7 @@ fn vec_dot_f32_f32_strided_simd(
 
 #[cfg(target_arch = "x86_64")]
 #[cfg(target_feature = "avx2")]
-fn vec_dot_f32_strided_simd(a: &[f32], a_base: usize, a_stride: usize, k: usize, b: &[f32]) -> f32 {
+fn vec_dot_f32_f32_strided_simd(a: &[f32], a_base: usize, a_stride: usize, k: usize, b: &[f32]) -> f32 {
     use std::arch::x86_64::*;
 
     unsafe {

